src/battle_game/core/README.md

```diff
@@ -24,12 +24,12 @@ protected:
 
 - game_core_
   - 这是一个指向游戏核心类的指针，会在对象被创建时赋值。当对象在运行过程中需要与游戏内其他元素交互时，
-你可以通过这个指针调用游戏核心的相应功能函数。
+    你可以通过这个指针调用游戏核心的相应功能函数。
```

Comment:
Lots of meaningless changes, plz fix them. (perhaps just use the current main-repo README.md here)